elide-model-config/src/main/resources/elideTableSchema.json

```diff
@@ -1,454 +1,426 @@
-          "default": "Prefab.Role.All"
-        },
-        "definition": {
-          "title": "Metric definition",
-          "description": "The definition of the metric",
-          "type": "string"
-        },
-        "queryPlanResolver": {
-          "title": "query plan resolver class",
-          "description": "The JVM function class which maps a metric to a query plan",
-          "pattern": "^([a-zA-Z]+[a-zA-Z0-9_]*\\.)+class",
-          "type": "string"
-        },
-        "type": { "$ref": "#/definitions/columnTypes" },
-        "tags": {
-          "title": "Measure tags",
-          "description": "An array of string based tags for measures",
-          "type": "array",
-          "uniqueItems": true,
-          "items": {
-            "type": "string"
-          },
-          "default": []
-        }
-      },
-      "required": [
-        "name",
-        "definition"
-      ],
-      "additionalProperties": false
-    },
-    "dimensionRef": {
-      "title": "Dimension",
-      "description": "Dimensions represent labels for measures. Dimensions are used to filter and group measures.",
-      "type": "object",
-      "properties": {
-        "name": {
-          "title": "Dimension name",
-          "description": "The name of the dimension. This will be the same as the POJO field name.",
-          "type": "string",
-          "pattern": "^[A-Za-z]([0-9A-Za-z]*_?[0-9A-Za-z]*)*$"
-        },
-        "description": {
-          "title": "Dimension description",
-          "description": "A long description of the dimension.",
-          "type": "string"
-        },
-        "category": {
-          "title": "Dimension group category",
-          "description": "Category for grouping dimension",
-          "type": "string"
-        },
-        "hidden": {
-          "title": "Hide/Show dimension",
-          "description": "Whether this dimension is exposed via API metadata",
-          "type": "boolean",
-          "default": false
-        },
-        "readAccess": {
-          "title": "Dimension read access",
-          "description": "Read permission for the dimension.",
-          "type": "string",
-          "default": "Prefab.Role.All"
-        },
-        "definition": {
-          "title": "Dimension definition",
-          "description": "The definition of the dimension",
-          "type": "string",
-          "default": ""
-        },
-        "cardinality": {
-          "$ref": "#/definitions/cardinalityTypes"
+    "definitions": {
+        "join": {
+            "title": "Join",
+            "description": "Joins describe the SQL expression necessary to join two physical tables. Joins can be used when defining dimension columns that reference other tables.",
+            "type": "object",
+            "properties": {
+                "name": {
+                    "title": "Join name",
+                    "description": "The name of the join relationship.",
+                    "type": "string"
```

Comment:
This should probably have a format restriction.

Reply:
added.